halyard-config/halyard-config.gradle

```diff
@@ -2,4 +2,7 @@ dependencies {
   compile spinnaker.dependency('bootActuator')
   compile spinnaker.dependency('bootWeb')
   compile spinnaker.dependency('lombok')
+
+  compile 'io.fabric8:kubernetes-api:2.2.161'
+  compile 'com.netflix.spinnaker.clouddriver:clouddriver-kubernetes:1.388.0'
```

Comment:
If there is a good reason to pin a version here, please leave a comment describing it.